floyd/client/data.py

```diff
@@ -74,7 +73,7 @@ def get_all(self):
 
     def delete(self, data_id):
         try:
-            self.request("DELETE", "%s/%s" % (self.url, data_id), timeout=10)
```

Comment:
Nice. It's funny what kind of wacky stuff we come up with when we're on autopilot.